src/executor/operator/physical_command.cppm

```diff
@@ -54,6 +55,8 @@ public:
     inline SharedPtr<Vector<SharedPtr<DataType>>> GetOutputTypes() const override { return output_types_; }
 
 public:
+    SharedPtr<BaseTableRef> table_ref_{}; // only used for compact command
```

Comment:
Can we move this memory to CommandInfo?

Reply:
CommandInfo is defined in parser module. which cannot access `BaseTableRef`